client/src/driver/BitBakeProjectScanner.ts

```diff
@@ -25,6 +25,7 @@ import fs from 'fs'
 import { runBitbakeTerminalCustomCommand } from '../ui/BitbakeTerminal'
 import { bitbakeESDKMode } from './BitbakeESDK'
 import { finishProcessExecution } from '../utils/ProcessUtils'
+import { extractRecipeName } from '../lib/src/utils/files'
```

Comment:
in the case where multiple versions exist and one is selected through PREFERRED_VERSION, we sometimes list the wrong .bb in the recipe view. Maybe this bug was present before your changes

Reply:
Can I have some examples?

Reply:
I'll create a separate ticket for this